lib/disk_utils.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+package disk_utils;
+use strict;
+use warnings;
+use testapi;
+use File::Spec::Functions 'catfile';
+use Test::Assert ':all';
+use Exporter 'import';
+our @EXPORT = qw(
+  get_disk_size
+  get_used_disk_space
+);
+
+sub df_command {
+    my $args = shift;
+    return script_output("df -h $args->{partition} | awk \'NR==2 {print \$$args->{column}}\'");
+}
+
+sub get_disk_size {
```

Comment:
Could you please write perl doc for those methods?

Reply:
I added description in "=head2...=cut" . Is that what you meant?